### PR TITLE
update cats-dogs training scripts

### PR DESCRIPTION
**Pull request by ntanganiya**
updated cats-dogs training scripts.

### CODE REVIEW

**Review by rotx-maxim — CHANGES_REQUESTED** [review]
Please revert the change to .python-version

File: datasets/cats_vs_dogs.py [hunk 1]
```diff
@@ -35,14 +35,17 @@ def catsdogs_get_datasets(data, load_train=True, load_test=True):
         is_dir = os.path.isdir(dataset_path)
 
         if not is_dir:
-            print("******************************************")
+            print("\n******************************************")
             print("Please follow instructions below:")
-            print("Download the dataset in the current working directory by visiting this link"
+            print("Download the dataset in the \'ai8x-training\' directory by visiting this link"
```
Comment (rotx-maxim) [inline datasets/cats_vs_dogs.py:40]: This should be \'ai8x-training/data\'

File: datasets/cats_vs_dogs.py [hunk 2]
```diff
@@ -35,14 +35,17 @@ def catsdogs_get_datasets(data, load_train=True, load_test=True):
...
                   "\'https://www.kaggle.com/c/dogs-vs-cats/data\'")
             print("and click the \'Download all\' button")
             print("If you do not have a Kaggle account, sign-up first.")
-            print("Unzip \'dogs-vs-cats.zip\' and you will see train.zip, test1.zip and .csv "
-                  " file. Unzip the train.zip file and re-run the script")
+            print("Unzip \'dogs-vs-cats.zip\' and you will see train.zip"
+                  " file. Unzip the train.zip file inside \'dogs-vs-cats\' directory.")
```
Comment (rotx-maxim) [inline datasets/cats_vs_dogs.py:45]: and this one is cats_vs_dogs

File: datasets/cats_vs_dogs.py [hunk 3]
```diff
@@ -35,14 +35,17 @@ def catsdogs_get_datasets(data, load_train=True, load_test=True):
...
+            print("This script splits the data from \'ai8x-training/dogs-vs-cats/train\'"
```
Comment (rotx-maxim) [inline datasets/cats_vs_dogs.py:46]: And the full path should be ai8x-training/data/cats_vs_dogs/train

**Review by rotx-maxim — CHANGES_REQUESTED** [review]

File: datasets/cats_vs_dogs.py (hunk 1 above)
Comment (ntanganiya) [reply datasets/cats_vs_dogs.py:40]: The script will take care of putting the dataset into 'ai8x-training/data' directory and remove the downloaded file from 'ai8x-training' directory.

File: datasets/cats_vs_dogs.py (hunk 2 above)
Comment (ntanganiya) [reply datasets/cats_vs_dogs.py:45]: The downloaded file from kaggle website is named 'dogs-vs-cats.zip'. After running this script, it's renamed as 'cats_vs_dogs'

File: datasets/cats_vs_dogs.py (hunk 3 above)
Comment (ntanganiya) [reply datasets/cats_vs_dogs.py:46]: The file is downloaded in 'ai8x-training' directory. Script will put files into right directory.

**Pull request closed without merging**